src/apps/backend/modules/organization-account/rest-api/organization-account-rest-api-server.ts

```diff
@@ -0,0 +1,23 @@
+import bodyParser from 'body-parser';
+import express, { Application } from 'express';
+
+import ErrorHandler from '../../error/error-handler';
+import OrganizationAccountRepository from '../internal/store/organization-account-repository';
+
+import OrganizationAccountRouter from './organization-account-router';
+
+export default class OrganizationAccountRESTApiServer {
+  public static async create(): Promise<Application> {
+    await OrganizationAccountRepository.createDBConnection();
+
+    const app = express();
+    app.use(bodyParser.urlencoded({ extended: true }));
+    app.use(bodyParser.json());
+
+    app.use('/organization account', OrganizationAccountRouter.getRoutes());
```

Comment:
can this be asked as a question too - like what rest base url you want for the module? If user presses Enter you can fall back to default one.

Also we are missing a hyphen